EIPS/eip-revert-specification.md

```diff
@@ -0,0 +1,78 @@
+---
```

Comment:
```suggestion
---
eip: 838
```
Assigning number 838 as documenting ERC838 from https://github.com/ethereum/EIPs/issues/838
Please also rename the filename

Reply:
Done